storehaus-algebra/src/main/scala/com/twitter/storehaus/algebra/MergeableStore.scala

```diff
@@ -32,6 +31,7 @@ object MergeableStore {
   implicit def enrich[K, V](store: MergeableStore[K, V]): EnrichedMergeableStore[K, V] =
     new EnrichedMergeableStore(store)
 
+  // todo(pankajg) After merging latest algebird replace this with Semigroup.maybePlus
```

Comment:
can we not use `todo` and instead file issues. We don't always close issues, but we approximately never address todos.